imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Query {
+  systemInformation: SystemInfo
+}
+
+"Represents Reaction Plugin"
+type Plugin {
+  name: String,
+  version: String
+}
+
+"Represents Reaction System Infomation"
+type SystemInfo {
```

Comment:
Might as well call this `SystemInformation`, too. Best to not abbreviate if we don't have to.